slurmweb/apps/agent.py

```diff
@@ -9,8 +9,24 @@
 import logging
 
 from rfl.web.tokens import RFLTokenizedRBACWebApp
-from racksdb.errors import RacksDBSchemaError, RacksDBFormatError
-from racksdb.web.app import RacksDBWebBlueprint
+
+try:
```

Comment:
Move the imports under the `if self.settings.racksdb.enabled`, similarly to what's done for metrics. This should save the extra logic with `RACKSDB_AVAILABLE` boolean and stub exceptions.